orangecontrib/text/widgets/owpreprocess.py

```diff
@@ -548,8 +553,8 @@ def export_data(self):
 
 PREPROCESSOR_MODULES = [
     TokenizerModule,
-    TransformationModule,
     CasingModule,
+    TransformationModule,
```

Comment:
Why this change? `TransformationModule` should be before `CasingModule` as is the case in GUI. This will reintroduce a bug (since `persistent_data` will not be in sync with `PREPROCESSOR_MODULES`) that was fixed [a few days ago](https://github.com/biolab/orange3-text/commit/f6d765dbe166d01a59dcf160d7c45a0b74a2a627).